src/View.elm

```diff
@@ -0,0 +1,178 @@
+    case ( confirmingDelete, errors, underEdit ) of
+        ( Just ( id, { firstName, lastName } ), _, _ ) ->
+            modalFooter [ css [ backgroundColor (hex "eaa") ] ]
+                [ text ("Do you want to delete " ++ firstName ++ " " ++ lastName ++ "? ")
+                , button [ onClick (DeleteUser id) ] [ text "yes" ]
+                , button [ onClick (IntendDeleteUser Nothing) ] [ text "no" ]
+                ]
+
+        ( Nothing, errs :: _, _ ) ->
+            footer [ css [ backgroundColor (hex "eaa") ] ]
+                [ text ("Error: " ++ String.join ", " errs)
+                , button [ onClick DismissLastError ] [ text "dismiss" ]
+                ]
+
+        ( Nothing, [], NewUser user ) ->
+            footer [ css [ backgroundColor (hex "aea") ] ]
+                [ button [ onClick (PostUser user) ] [ text "save" ]
+                , button [ onClick (Edit NotEditing) ] [ text "discard" ]
+                ]
+
+        ( Nothing, [], Id id user ) ->
+            footer [ css [ backgroundColor (hex "eea") ] ]
+                [ button [ onClick (PutUser id user) ] [ text "save" ]
+                , button [ onClick (Edit NotEditing) ] [ text "discard" ]
+                , button [ onClick (IntendDeleteUser (Just ( id, user ))) ] [ text "delete" ]
+                ]
+
+        _ ->
+            footer [] []
+
+
+viewCreateUserMedallion : UnderEdit -> Html Msg
+viewCreateUserMedallion underEdit =
+    case underEdit of
+        NewUser user ->
+            medallion [ css [ backgroundColor (hex "aea") ] ] (inputs user (Edit << NewUser))
+
+        _ ->
+            medallion [ css [ cursor pointer ], onClick (Edit (NewUser initUser)) ] [ span [ css [ fontSize (rem 2) ] ] [ text "➕" ] ]
+
+
+viewUser : (List (Attribute Msg) -> List (Html Msg) -> Html Msg) -> UnderEdit -> ( Int, User ) -> Html Msg
+viewUser wrapper underEdit ( id, { firstName, lastName, elmSkill } as user ) =
+    let
+        default =
+            wrapper [ css [ cursor pointer ], onClick (Edit (Id id user)) ] (texts user)
+    in
+    case underEdit of
+        Id id_ user_ ->
+            if id == id_ then
+                wrapper [ css [ backgroundColor (hex "eea") ] ] (inputs user_ (Edit << Id id))
+
+            else
+                default
+
+        _ ->
+            default
+
+
+medallion : List (Attribute msg) -> List (Html msg) -> Html msg
+medallion extraAttrs =
+    div
+        (css
+            [ borderRadius (pct 50)
+            , border2 (px 1) solid
+            , displayFlex
+            , flexDirection column
+            , alignItems center
+            , justifyContent center
+            , property "align-content" "center"
+            , property "aspect-ratio" "1"
+            ]
+            :: extraAttrs
+        )
+
+
+row : List (Attribute msg) -> List (Html msg) -> Html msg
+row extraAttrs =
+    div (css [ borderBottom2 (px 1) solid, displayFlex, flexDirection column, padding (px 2) ] :: extraAttrs)
```

Comment:
I am inclined to think the terser the code, the more surplus enjoyment the writer takes from the future reader. And I've even cut you some slack here :D Bite on some wood now because I've considered writing:
```
row =
    div << (::) (css [ borderBottom2 (px 1) solid, displayFlex, flexDirection column, padding (px 2) ])
```

Reply:
You're my man 😂 I have countless `CR fix` commits with removing point free, but on the other hand with elm-format we have most of the code multi-lined and for me personally it's totally fine (I'm coding on 13" laptop screen, so don't have that much horizontal space).